src/components/Button/Button.module.css

```diff
@@ -18,3 +17,44 @@
 .typeSecondary:hover {
     background-color: rgba(67, 90, 111, 0.06);
 }
+
+.typePrimary:focus {
+  outline: none;
+  box-shadow: 0 0 0 2px var(--color-primary-contrast);
+}
+
+.typeSecondary:focus {
+  outline: none;
+  box-shadow: 0 0 0 2px var(--color-primary-contrast);
+}
```

Comment:
This selector has the same css props as `.typePrimary:focus`. Refactor  `.typePrimary:focus` to `.base:focus` and remove `.typeSecondary:focus`?

Reply:
Right!! This was a copy paste error. Should have different color variable